src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -84,4 +100,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     response.sendRedirect("/user-page.html?user=" + recipient);
   }
+
+  // Translates text into target language
```

Comment:
While it is not required for private functions, if you want to put a function level comment here, lets use Javadoc styling.

/** Translates text into target language. */